tests/xharness/Xharness.Tests/Tests/AppRunnerTests.cs

```diff
@@ -709,19 +703,19 @@ public async Task RunOnDeviceWithFailedTestsTest ()
 					ips.Append (',');
 				ips.Append (ipAddresses [i].ToString ());
 			}
-
-			var expectedArgs = $"--sdkroot {xcodePath} -v -v -argument=-connection-mode -argument=none " +
+			
+			var expectedArgs = $"--sdkroot {StringUtils.FormatArguments (xcodePath)} -v -v -argument=-connection-mode -argument=none " +
 				$"-argument=-app-arg:-autostart -setenv=NUNIT_AUTOSTART=true -argument=-app-arg:-autoexit " +
 				$"-setenv=NUNIT_AUTOEXIT=true -argument=-app-arg:-enablenetwork -setenv=NUNIT_ENABLE_NETWORK=true " +
 				$"-setenv=DISABLE_SYSTEM_PERMISSION_TESTS=1 -argument=-app-arg:-hostname:{ips} -setenv=NUNIT_HOSTNAME={ips} " +
 				$"-argument=-app-arg:-transport:Tcp -setenv=NUNIT_TRANSPORT=TCP -argument=-app-arg:-hostport:{simpleListener.Object.Port} " +
 				$"-setenv=NUNIT_HOSTPORT={simpleListener.Object.Port} -setenv=env1=value1 -setenv=env2=value2 " +
-				$"--launchdev {appPath} --disable-memory-limits --wait-for-exit --devname Test iPad";
+				$"--launchdev {StringUtils.FormatArguments (appPath)} --disable-memory-limits --wait-for-exit --devname \"Test iPad\"";
 
 			processManager
 				.Setup (x => x.ExecuteCommandAsync (
 					mlaunchPath,
-					It.Is<IList<string>> (args => string.Join (" ", args) == expectedArgs),
+					It.Is<MlaunchArguments> (args => args.AsCommandLine() == expectedArgs),
```

Comment:
space after `(`

Reply:
I have ran the auto-formatter on the files where code style was off